fabric-item-api-v1/src/main/java/net/fabricmc/fabric/mixin/item/ItemStackMixin.java

```diff
@@ -100,4 +101,14 @@ public Multimap<EntityAttribute, EntityAttributeModifier> hookGetAttributeModifi
 	public boolean hookIsSuitableFor(Item item, BlockState state) {
 		return item.isSuitableFor((ItemStack) (Object) this, state);
 	}
+
+	/**
+	 * @author Phoupraw
+	 * @reason use stack-aware {@link #getFoodComponent()} instead
+	 */
+	@Overwrite
```

Comment:
We dont use `@Overwrite` in fabric api

Reply:
Always use `@Inject` and `cancel` at `@At("RETURN")`?